src/rgw/rgw_sal_motr.cc

```diff
@@ -898,7 +898,8 @@ int MotrBucket::merge_and_store_attrs(const DoutPrefixProvider *dpp, Attrs& new_
   for (auto& it : new_attrs)
     attrs[it.first] = it.second;
 
-  return put_info(dpp, y, ceph::real_time());
+  // Call put_info with false to update the attributes
```

Comment:
Update the comment as :
"merge_and_store_attrs" meant for updating existing metadata, hence explicitly passing false. Passing "yield" was incorrect

Reply:
Done

Reply:
@sachinpunadikar doesnt look right, why we need to mention merge_and_store_attrs() here ?